front/src/modules/people/components/PersonChip.tsx

```diff
@@ -3,6 +3,8 @@ import { Link } from 'react-router-dom';
 import { Theme } from '@emotion/react';
 import styled from '@emotion/styled';
 
+import { Avatar } from '@/users/components/Avatar';
+
 import PersonPlaceholder from './person-placeholder.png';
```

Comment:
(github action alert: PersonPlaceholder' is defined but never used)

Reply:
Fixed 👍